src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useContext, useState } from 'react';
-import { Text, View, TouchableOpacity, Image, ScrollView } from 'react-native';
+import { Text, View, TouchableOpacity, Image, ScrollView, ActivityIndicator } from 'react-native';
```

Comment:
sure I will do that

Reply:
ETA please ?

Reply:
Today EOD

Reply:
@shreya-mishra if we use loader component then we need to make a lot of changes in that so it might create a problem when we want to use it in any other file